lms/djangoapps/teams/templates/teams/teams.html

```diff
@@ -11,14 +12,14 @@
 
 <%include file="/courseware/course_navigation.html" args="active_page='teams'" />
 
+<div class="team-tab-header"></div>
 <div class="team-tab-content"></div>
 
 <%block name="js_extra">
 <script type="text/javascript">
     (function (require) {
         require(['teams/js/teams_tab_factory'], function (TeamsTabFactory) {
-            var pageView = new TeamsTabFactory({
-            });
+            var pageView = new TeamsTabFactory(${json.dumps(header_info)});
```

Comment:
I think I wrote this, but I think it might be cleaner if we pass the root element to the factory, rather than assuming its existence. In general, I like Backbone views that can be constructed with no assumptions about where they will be rendered. If no element is passed to them, they create their own element, and then the view can be appended into the DOM. If instead an element is passed in, then they use that as their root.